src/main/java/frc/trigon/robot/RobotContainer.java

```diff
@@ -71,7 +97,7 @@ private void bindCommands() {
     private void putSendablesOnSmartDashboard() {
         SmartDashboard.putData(Shooter.getInstance());
         SmartDashboard.putData(Pitcher.getInstance());
-        SmartDashboard.putData(limelight);
+        SmartDashboard.putData("Hub Limelight", limelight);
```

Comment:
Thats why I couldn't find "distance from hub" on the dashboard

Reply:
Can I resolve this?